django_pain/management/commands/import_payments.py

```diff
@@ -0,0 +1,66 @@
+"""Command for importing payments from bank."""
+import importlib
+import sys
+from typing import Sequence
+
+from django.core.exceptions import ValidationError
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+
+from django_pain.models import BankAccount
+
+
+class Command(BaseCommand):
+    """Import payments from bank."""
+
+    help = 'Import payments from the bank. Bank statement should be provided on standard input.'
+
+    def add_arguments(self, parser):
+        """Command takes one argument - dotted path to parser class."""
+        parser.add_argument('parser', type=str, help='dotted path to parser class')
+
+    @staticmethod
+    def get_parser(parser_path):
+        """Get parser instance from parser path argument."""
+        module_name, class_name = parser_path.rsplit('.', 1)
+
+        parser_module = importlib.import_module(module_name)
+        parser_class = getattr(parser_module, class_name)
+        return parser_class()
```

Comment:
Seems to duplicate `django.utils.module_loading.import_string`.

Reply:
Nice catch! Fixed.